pages/common/virt-sysprep.md

```diff
@@ -0,0 +1,20 @@
+# virt-sysprep
+
+> Reset, unconfigure, or customize a virtual machine image.
+> More information: <https://manned.org/virt-sysprep.1>.
+
+- List all supported operations (enabled operations are indicated with asterisks):
+
+`virt-sysprep --list-operations`
+
+- Run the enabled operations but don't apply the changes:
+
+`virt-sysprep --domain {{vm_name}} --dry-run`
+
+- Run the specified operations instead of the enabled operations:
+
+`virt-sysprep --domain {{vm_name}} --operations {{operation, operation, ...}}`
```

Comment:
```suggestion
`virt-sysprep --domain {{vm_name}} --operations {{operation1, operation2, ...}}`
```
@bl-ue the commas are not part of the arguments itself, but a separator. Should we write `{{operation1}}, {{operation2}}, {{...}}` in that case?

Reply:
Hmm.  I don't think so—as-is it seems fine to me.  Am I missing something?

Reply:
No, I was just wondering if we should consider the `,` part of the variable or not.

Reply:
Shouldn't we? Because if there's only one operation specified, isn't there no comma?

Reply:
A comma that isn't followed by an operation throws this error:

virt-sysprep: error: --operations: empty operation name.

I've removed the spaces after the commas in the operations list because they also cause the error to be thrown.

Reply:
Do I need to make any other changes to this example?